system/forms/formbuilder/exportConfiguration.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--#!autodoc
+Data export: configuration form
```

Comment:
I think this documentation comment needs updating to be specific to form builder?